### DISCUSSION TITLE
How can I take user arguments?

### DISCUSSION THREAD

**Discussion by tsklxiv** [janet-lang/janet#706] [Q&A] [answered]
Hello again.

Currently I am developing a simple RPN calculator written in Janet currently called `jncalc`. It is heavily inspired by GNU's `dc` with some changes. Now everything in quite finished and I want to do one thing is that I want to take user arguments when they run the program (e.g. `janet jncalc.janet --help`, `--help` is an user argument). I don't find anything related to this process on the Janet Documentation, so I'm asking for it.

**Comment by pepe** [comment, marked as answer]
Hello. `argparse` is your friend for this: https://github.com/janet-lang/spork#argparse.
